### PR TITLE
NMS-7986: Documentation how to use RRDtool

### PR DESCRIPTION
**Pull request by indigo423**
Created documentation how to install RRDtool and how to configure OpenNMS to enable it. The documentation covers the installation on CentOS/RHEL and Debian/Ubuntu based operation systems.

JIRA: http://issues.opennms.org/browse/NMS-7986
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS279

### CODE REVIEW

**Comment by mvrueden** [comment]
I could not find documentation for Windows. Isn't it possible to run rrdtool under Windows?

**Comment by indigo423** [comment]
I need more time for investigation. I don't know at the current point in time. Similar to R. I do not want to block people using it because most of users run on Linux.

File: opennms-doc/guide-install/src/asciidoc/text/rrdtool/installation.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+[[gi-rrdtool-configure-opennms]]
+=== Configuration of OpenNMS
+
+To configure _OpenNMS_ to use _RRDtool_ instead of _JRobin_ configure the following properties in `rrd-configuration.properties`.
+
+.Configuration of RRDtool in OpenNMS on RHEL/CentOS
+[source]
+----
+org.opennms.rrd.strategyClass=org.opennms.netmgt.rrd.rrdtool.MultithreadedJniRrdStrategy
+org.opennms.rrd.interfaceJar=/usr/share/java/jrrd2-api-2.0.1.jar
```
Comment (j-white) [inline opennms-doc/guide-install/src/asciidoc/text/rrdtool/installation.adoc]: This is currently accurate, but once NMS-7907 is fixed, the path to the .jar will be:
/usr/share/java/jrrd2.jar

Comment (indigo423) [inline opennms-doc/guide-install/src/asciidoc/text/rrdtool/installation.adoc]: Good hint - I'll wait NMS-7907 is fixed and update accordingly, we can merge it afterwards.

Comment (j-white) [inline opennms-doc/guide-install/src/asciidoc/text/rrdtool/installation.adoc]: NMS-7907 is fixed now

**Pull request merged**